source/policies.ttl

```diff
@@ -0,0 +1,17 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix : <https://id.kb.se/vocab/>
+base <https://id.kb.se/>
+
+</policy/freely-available> a :AccessPolicy ;
+    skos:prefLabel "Fritt tillgänglig"@sv, "Freely available"@en .
+
+</policy/gratis> a :AccessPolicy ;
+    skos:prefLabel "gratis"@sv, "gratis"@en ;
```

Comment:
```suggestion
    skos:prefLabel "kostnadsfri"@sv, "gratis"@en ;
```
Thoughts/suggestions?

Reply:
It might be important to specify OA context somewhere? Since it means "Öppen tillgång" or "Ej öppen tillgång". I'm gonna look around to see if we can find a preferred definition for this.

Reply:
Suggestion from the Swepub team applied # 9923ae92a9abd6a705c67a641cdf11544c58d69b